Configuration-Interaction/CI_DL.py

```diff
@@ -117,20 +112,14 @@
 for CI_ITER in range(max_guess - 1):
 
     # Subspace Matrix, Gij = < bi | H | bj >
-    #for i in range(num_vecs - num_eig, num_vecs):
     for i in range(0, num_vecs):
         # Build sigma for each b
         ciwfn.sigma(cvecs, svecs, i, i)
```

Comment:
Do `cvecs` and `svecs` have the right number of vectors to write into? You can also try a `cvecs.read(i, 0)` and `svecs.read(i, 0)` before hand. There might be some issue with it not paying attention to your `i` root.

Anything stand out about the resulting vector? Is it totally wrong, a different root, etc?